governance/polls/templates/Generic/Atlas Edit Weekly Cycle Proposal - September 16, 2024.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Atlas Edit Weekly Cycle Proposal - September 16, 2024
+summary: Signal your support or opposition to this Atlas edit, which: 1) Updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs; 2) adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators; 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2.
```

Comment:
The text in the summary causes a conflict due to the colons after the word **'which:'**. I suggest placing the entire text in quotation marks to resolve the issue without modifying the summary text.
```suggestion
summary: "Signal your support or opposition to this Atlas edit, which: 1) Updates the GSM Delay Exception for the Smart Burn Engine based on a request from BA Labs; 2) adds exemption for the Facilitator anonymity requirement for Ecosystem based on discussion with Governance Facilitators; 3) makes minor updates to language in some places to incorporate initial feedback on Atlas v2."
```

Reply:
We can also just remove the conflicting signs, no?